src/test/java/com/google/firebase/auth/FirebaseUserManagerTest.java

```diff
@@ -78,11 +82,13 @@
           .setAndroidInstallApp(true)
           .setAndroidMinimumVersion("6")
           .build();
+
   private static final Map<String, Object> ACTION_CODE_SETTINGS_MAP =
           ACTION_CODE_SETTINGS.getProperties();
 
   private static final String PROJECT_BASE_URL =
       "https://identitytoolkit.googleapis.com/v2/projects/test-project-id";
+
   private static final String TENANTS_BASE_URL = PROJECT_BASE_URL + "/tenants";
```

Comment:
Is `TENANTS_BASE_URL` still needed?

Reply:
There are tests in this file that makes tenant-aware user management calls.